src/test/java/wooteco/subway/AcceptanceTest.java

```diff
@@ -1,20 +1,105 @@
 package wooteco.subway;
 
 import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
 import org.junit.jupiter.api.BeforeEach;
 import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.boot.web.server.LocalServerPort;
-import org.springframework.test.annotation.DirtiesContext;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.jdbc.Sql;
+import wooteco.subway.line.dto.LineRequest;
+import wooteco.subway.section.dto.SectionRequest;
+import wooteco.subway.station.dto.StationRequest;
 
-@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+@Sql("/truncate.sql")
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 public class AcceptanceTest {
 
     @LocalServerPort
     int port;
 
+    protected ExtractableResponse<Response> 강남역_response;
+    protected ExtractableResponse<Response> 역삼역_response;
+    protected ExtractableResponse<Response> 도곡역_response;
+
     @BeforeEach
     public void setUp() {
         RestAssured.port = port;
+        강남역_response = postStation("강남역");
+        역삼역_response = postStation("역삼역");
+        도곡역_response = postStation("도곡역");
+    }
+
+    protected ExtractableResponse<Response> getStations() {
+        return RestAssured.given().log().all()
+            .when()
+            .get("/stations")
+            .then().log().all()
+            .extract();
+    }
+
+    protected ExtractableResponse<Response> postStation(String name) {
```

Comment:
```suggestion
    protected <T> ExtractableResponse<Response> post(String path, T requestBody) {
        return RestAssured.given().log().all()
                .body(requestBody)
                .contentType(MediaType.APPLICATION_JSON_VALUE)
                .when()
                .post(path)
                .then().log().all()
                .extract();
    }
```
제네릭을 사용하면 좀 더 범용적으로 쓸 수 있겠죠
path를 미리 고정해두면 계속 비슷한 형태의 메서드가 추가될거에요